articles/iot-edge/how-to-configure-api-proxy-module.md

```diff
@@ -16,15 +16,13 @@ monikerRange: ">=iotedge-2020-11"
 
 [!INCLUDE [iot-edge-version-202011](../../includes/iot-edge-version-202011.md)]
 
-The API proxy module enables IoT Edge devices to send HTTP requests through gateways instead of making direct connections to cloud services. This article walks through the configuration options so that you can customize the module to support your gateway hierarchy requirements.
+There are several scenarios where you might want to deploy multiple services on an IoT Edge device, all of which support HTTPS protocol and bind to port 443. This is especially relevant in hierarchical deployments of IoT Edge devices in ISA-95 based network isolated architectures such as those described [here](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-connect-downstream-iot-edge-device?view=iotedge-2020-11&tabs=azure-portal#network-isolate-downstream-devices), as the clients on the child devices are not able to connect to the cloud directly. For example, to allow child IoT Edge devices to pull docker images requires deploying a Docker registry module, and to allow uploading blobs requires deploying a Blob Storage module on the same IoT Edge device. Both these services use HTTPS for communication. The API Proxy enables such deployments on an IoT Edge device. Instead of each service, the API Proxy module binds to port 443 on the host device and routes the request to the right service module running on that device as per user configurable rules. The individual services are still responsible for handling the requests, including authenticating and authorization of the clients. Without the API Proxy, each service module would have to bind to a separate port on the host device, requiring tedious and error prone configuration change on each child device connecting to the parent IoT Edge device. 
```

Comment:
```suggestion
There are several scenarios where you might want to deploy multiple services on an IoT Edge device, all of which support HTTPS protocol and bind to port 443. This is especially relevant in hierarchical deployments of IoT Edge devices in ISA-95 based network isolated architectures such as those described in [Network isolate downstream devices](how-to-connect-downstream-iot-edge-device.md#network-isolate-downstream-devices), as the clients on the child devices are not able to connect to the cloud directly. For example, to allow child IoT Edge devices to pull Docker images requires deploying a Docker registry module, and to allow uploading blobs requires deploying a Blob Storage module on the same IoT Edge device. Both these services use HTTPS for communication. The API Proxy enables such deployments on an IoT Edge device. Instead of each service, the API Proxy module binds to port 443 on the host device and routes the request to the right service module running on that device as per user configurable rules. The individual services are still responsible for handling the requests, including authenticating and authorization of the clients. Without the API Proxy, each service module would have to bind to a separate port on the host device, requiring tedious and error prone configuration change on each child device connecting to the parent IoT Edge device. 
```

Reply:
I replaced the link text "Here" with the name of the section, and shortened the link target to be a relative filepath instead of absolute URL